src/test/java/com/wikia/webdriver/testcases/mediatests/addvideo/VetSpecialVideosTests.java

```diff
@@ -27,7 +27,7 @@
   @RelatedIssue(issueID = "MAIN-6406", comment = "Please, test manually by clicking the delete (bin) icon")
   @Execute(asUser = User.USER)
   public void SpecialVideos_001_Provider() {
-    String wikiURL = urlBuilder.getUrlForWiki("mobileregressiontesting");
+ //   String wikiURL = urlBuilder.getUrlForWiki("mobileregressiontesting");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)